openmdao/vectors/petsc_vector.py

```diff
@@ -107,9 +115,16 @@ def _initialize_data(self, root_vector):
         super(PETScVector, self)._initialize_data(root_vector)
 
         self._petsc = {}
+        self._imag_petsc = {}
```

Comment:
I am not sure. Does setting `.data` on the petsc object work collectively for distributed vectors? I think i'd rather put up a story to investigate this.